### ISSUE TITLE
Add support for Terraform v0.12

### ISSUE THREAD

**Issue by aaron-lane** [terraform-google-modules/terraform-google-network#43]

**Comment by btorresgil** [comment]
Error when using module v0.8.0 on Terraform v0.12:

```
$ terraform plan

Error: Unsupported block type

  on .terraform/modules/main.network_memory_store/terraform-google-modules-terraform-google-network-d1b6646/main.tf line 94, in resource "null_resource" "delete_default_internet_gateway_routes":
  94:   triggers {

Blocks of type "triggers" are not expected here. Did you mean to define
argument "triggers"? If so, use the equals sign to assign it a value.

```

**Comment by skenmy** [comment]
@aaron-lane is this issue resolved by #48 ?

**Comment by morgante** [comment]
Indeed it is.
